packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm/DataAccessRequestAccessorsFilesForm.test.tsx

```diff
@@ -347,14 +342,17 @@ describe('DataAccessRequestAccessorsFilesForm tests', () => {
       ),
     )
 
-    await expect(renderComponent(defaultProps)).rejects.toThrow(errorMessage)
```

Comment:
Sometimes this didn't throw in time for the `expect` to catch it. It seemed tricky to write code to catch an error thrown asynchronously after the component initially rendered, so instead just check that the error boundary shows the error message.